Remotion/ObjectBinding/Web/UI/Controls/ErrorColumnWithoutValidationFailureBehaviour.cs

```diff
@@ -0,0 +1,9 @@
+﻿namespace Remotion.ObjectBinding.Web.UI.Controls
+{
+  public enum ErrorColumnWithoutValidationFailureBehaviour
+  {
+    HideUnlessAnyValidationFailures,
+    HideUnlessRowOrCellValidationFailures,
+    Show
```

Comment:
The double negative/inversion (`Hide` + `Unless`) makes it hard to understand the names without docs. But it is kind of hard to find good names here... Maybe we discuss this separately and try to find good names together.

Reply:
Possible dev jour fixe thing? As we have @MichaelKetting there as well?

Reply:
Just renamed them from `HideUnless` to `DisplayIf` -> IMO easier to understand